src/Protocol/Diaspora.php

```diff
@@ -1885,14 +1890,15 @@ private static function receiveMessage(array $importer, $data)
 	/**
 	 * Processes participations - unsupported by now
 	 *
-	 * @param array  $importer Array of the importer user
-	 * @param object $data     The message object
+	 * @param array  $importer  Array of the importer user
+	 * @param object $data      The message object
+	 * @param int    $direction Indicates if the message had been fetched or pushed
```

Comment:
Please add a reference to the constants the parameter is taking its values from.